ambry-router/src/main/java/com.github.ambry.router/TtlUpdateOperation.java

```diff
@@ -51,7 +52,7 @@
   // The operation tracker that tracks the state of this operation.
   private final OperationTracker operationTracker;
   // A map used to find inflight requests using a correlation id.
-  private final Map<Integer, TtlUpdateRequestInfo> ttlUpdateRequestInfos = new HashMap<>();
+  private final Map<Integer, TtlUpdateRequestInfo> ttlUpdateRequestInfos = new TreeMap<>();
```

Comment:
why this change?

Reply:
In fetchRequests function, this map will be iterated based on the order of it's key. So I change it to a sorted tree.